ETC.tsv

```diff
@@ -13345,67 +13345,65 @@ ETC_20150717_013345	button enables you to reset all items registered for Buy and
+ETC_20150717_013353	When enhancing +4 weapons and above, Potential will be consumed on failure. The item will be destroyed when Potential falls below zero.
+ETC_20150717_013354	Enhancement penalties begin above +2 for armors, bracelets, and necklaces.
 ETC_20150717_013355	You can't use an item with 0 durability.
-ETC_20150717_013356	"You can repair item with low durability through the Repair window.{nl}Select item then click Repair and pay the fee to restore the item's durability to max. 
-"
-ETC_20150717_013357	"A repair fee will be charged. Higher ranked items are usually more expensive to repair.
-"
+ETC_20150717_013356	Repair items in the Repair window.{nl}Select an item, then click Repair to restore durability to 100%.
+ETC_20150717_013357	A repair fee will be charged. Higher ranked items are usually more expensive to repair.
 ETC_20150717_013358	Use Gems on your items for additional effects.
-ETC_20150717_013359	Gems can be obtained from dugeons for additional item effects.
-ETC_20150717_013360	Gems can be upgraded through Gem Enhancement. Gem Abrasives or other Gems are required to Enhance Gems.
-ETC_20150717_013361	Insert Gems in the item sockets to add item effects.
+ETC_20150717_013359	Gems used for extra item effects can be obtained from dungeons.
+ETC_20150717_013360	Gems can be upgraded through Gem Enhancement. Gem Abrasives or other Gems are required.
+ETC_20150717_013361	Insert Gems in the item sockets to add additional effects.
 ETC_20150717_013362	Some items may not have sockets.
-ETC_20150717_013363	For such items, you can add sockets through the Blacksmith NPC.
-ETC_20150717_013364	Adding sockets will require socket processing fees and the item's potential will be reduced by 1.
-ETC_20150717_013365	{img mouseclick_right 40 40} on the Gem to see in which items the Gem can be inserted.
+ETC_20150717_013363	You can add Gem sockets at the Blacksmith NPC.
+ETC_20150717_013364	Adding sockets will consume a fee and reduce the item's Potential by 1.
+ETC_20150717_013365	Press {img mouseclick_right 40 40} on the Gem to see in which items are compatible.
 ETC_20150717_013366	Click on the item to insert the Gem.
-ETC_20150717_013367	You can check the inserted Gems through in the item's tooltips.
+ETC_20150717_013367	Hover the cursor over an item to view its current Gems.
 ETC_20150717_013368	You can remove inserted Gems by
 ETC_20150717_013369	through the Blacksmith NPC.
 ETC_20150717_013370	The Gem's level will be lowered by 1 when removed.
-ETC_20150717_013371	SubWeapon refers to the items equipped on Character's right hand.
-ETC_20150717_013372	When SubWeapon is equipped, press {img C 40 40} key to use the it.
-ETC_20150717_013373	Your character will use the SubWeapon to attack.
-ETC_20150717_013374	If the SubWeapon has special effects, weapon effects will be triggered.
-ETC_20150717_013375	Trade between players
+ETC_20150717_013371	A Sub-Weapon is an item in the Character's right hand.
```

Comment:
A Sub-Weapon is an item in the Character's right hand. **->** The sub-weapon is a secondary weapon equippable in the character's right hand.